includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -285,15 +299,6 @@ private function getVerticalsSection() {
 			'links' => [
 				[
 					'type' => 'link-branded',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "link-branded" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)